### ISSUE TITLE
xcrun: error: invalid active developer path 

### ISSUE THREAD

**Issue by ABevier** [teaxyz/cli#551]
Tea is failing to sync pantries with the following error:

```
[2023-04-25 01:23:17.194] [error] Error: Command failed: cd '/Users/redacted/.tea/tea.xyz/gui' && ./tea -S
tea: inspecting pantries…
^[[G^[[1A^[[0Jtea: fetching pantries…

xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun


panic: spilt tea. we're sorry and we'll fix it… but you have to report the bug!

    https://github.com/teaxyz/cli/issues/new?title=panic%3Acmd+failed%3A+1%3A+%2Fusr%2Fbin%2Fgit+clone+--bare+--depth%3D1+https%3A%2F%2Fgithub.com%2Fteaxyz%2Fpantry+%2FUsers%2Fredacted%2F.tea%2Ftea.xyz%2Fvar%2Fpantries%2Fteaxyz%2Fpantry

----------------------------------------------------->> attachment begin
Error: cmd failed: 1: /usr/bin/git clone --bare --depth=1 https://github.com/teaxyz/pantry /Users/redacted/.tea/tea.xyz/var/pantries/teaxyz/pantry
    at useRun (file:///Users/builder/actions-runner/_work/cli/cli/src/hooks/useRun.ts:39:34)
    at async clone (file:///Users/builder/actions-runner/_work/cli/cli/src/hooks/useSync.ts:101:9)
    at async file:///Users/builder/actions-runner/_work/cli/cli/src/hooks/useSync.ts:80:17
```

**Comment by mxcl** [comment]
Notable missing detail in the report is the absense of Xcode or Xcode CLT.

**Comment by mxcl** [comment]
k lol it's a broken xcode command line tools installation. We can detect this and then use our own git.

I think what I'm going to do is always http fetch the pantry and then convert to a git install installing our own git as necessary

**Comment by mxcl** [comment]
Fixed in #555
